api/compile.sh

```diff
@@ -1,23 +1,23 @@
 #!/bin/bash
-set -eux
```

Comment:
This is always good to have.
For backgound, see; http://redsymbol.net/articles/unofficial-bash-strict-mode/